src/app/pages/cases/details/DetailsPage.tsx

```diff
@@ -50,7 +53,7 @@ const DetailsPage: React.FC<RouteComponentProps<Props>> = ({ id: idString }) =>
       </DefaultLayout> :
     showNotFound ?
       <NotFoundPage /> :
-      <PageSpinner />
+      null
```

Comment:
Shouldn't this show some kind of errormessage?

Reply:
Yes, maybe, but it's a situation that can not exist actually. I'll use `&&` to make it cleaner.